kii/kii.h

```diff
@@ -36,6 +36,8 @@ extern "C" {
 #define M_KII_LOG(x)
 #endif
 
+#define KII_PUSH_KEEP_ALIVE_INTERVAL_SECONDS 30
```

Comment:
@HongweiGuo 
Current Guidance is based on the assumption that the macro is definedin build system (ex. Makefile) or application side.

Is it possible to define this macro in your build system?